packages/expo-video/src/VideoView.web.tsx

```diff
@@ -16,8 +23,13 @@ if (audioContext && zeroGainNode) {
     "Couldn't create AudioContext, this might affect the audio playback when using multiple video views with the same player."
   );
 }
-class VideoPlayerWeb implements VideoPlayer {
+
+class VideoPlayerWeb
+  extends globalThis.expo.SharedObject<VideoPlayerEvents>
```

Comment:
The goal is to get rid of referencing `globalThis.expo` except files that are intended to export it, i.e. `EventEmitter.ts`, `SharedObject.ts` and so on.
For now it's fine, I'll clean this up soon.